src/common/config/appearance-options.ts

```diff
@@ -13,6 +13,10 @@ export interface AppearanceOptions {
     windowWidth: number;
     allowTransparentBackground: boolean;
     fontFamily: string;
+    userInputRadius: string;
+    userInputBottomMargin: number;
+    searchResultsRadius: string;
+    scrollbarRadius: string;
```

Comment:
How about renaming `userInputRadius` to `userInputBorderRadius`, then I think it's more clear what it is. Also the other variables which contain `Radius`.